Grid layer. (#524)
ElektraInitiative/PermaplanT: doc/usecases/done/gridlayer.md

> # Use Case: Grid Layer

Reviewer: I don't think that a layer is right, layers are only useful if some elements can be associated to it. In case of grid I would simply add a left toolbar, like for measurement.

Author: I wasn't sure how to call this feature and I agree that layer is probably the wrong category.

We should certainly agree on a term for Konva-Layers that are not tied to the backend in some way, since other use cases (e.g. the base layer for measuring distances) will have to implement them as well.
One candidate I thought of would be "pseudo-layer", but that might be a little confusing.
"Overlay" might be another option.

Reviewer: That a konvas layer is used is purely an implementation detail, it is not the use case concern.

```suggestion
# Use Case: Grid
```

Author: > That a konvas layer is used is purely an implementation detail, it is not the use case's concern.

Sorry, I didn't phrase that correctly. I meant layers that only exist in the frontend.

## Summary

> - **Scope:** Grid Layer

Reviewer: ```suggestion
- **Scope:** Grid
```

- **Level:** User Goal
- **Actors:** App User
- **Brief:** The app will display a fixed scale coordinate grid.
- **Status:** Done
- **Assignee:** Moritz

## Scenarios

- **Precondition:**
    - User has opened the map.
- **Main success scenario:**

>     - The user sees a fixed scale coordinate grid with 10 centimeter spacing.

Reviewer: 10 centimeter is too fine scaled.

```suggestion
    - The user sees a fixed scale coordinate grid with 1 meter spacing.
```

>     - Dots that indicate 10 m are bigger.

Reviewer: Doesn't seem to be implemented.

```suggestion
```

>     - Lines that are a whole number of meters away from the origin are drawn boldly.

Reviewer: Only nice to have:

```suggestion
    - Dots that indicate 10 m are bigger.
```

- **Non-functional Constraints:**
    - Support for changing viewport (zoom, position, etc.)

>     - The functionality is only available in the frontend.

Reviewer: Links to implementation, protocol, ... missing

Author: Just for clarification: are we writing a test protocol for each PR now?

What do you mean by linking the implementation? 
Do you want a link to the folder where the feature is implemented?

>     - The grid layer is only available in the frontend.

Reviewer: ```suggestion
    - The functionality is only available in the frontend.
```
